lib/passport-mock/strategy.js

```diff
@@ -40,11 +49,24 @@ MockStrategy.prototype.authenticate = function authenticate (req, options) {
         delete this.__proto__._accessToken;
         delete this.__proto__._refreshToken;
         delete this.__proto__._profile;
+        delete this.__proto__._expires_in;
+        delete this.__proto__._scope;
+        delete this.__proto__._token_type;
       };
 
-      this.verify(accessToken, refreshToken, profile, function (err, user) {
-        this.success(user);
-      }.bind(this));
+      var arity = this.verify.length;
+      if (arity === 6) {
+        this.verify(req, accessToken, refreshToken, params, profile, verified);
+      } else if (arity === 5) {
+        if (this._passReqToCallback) {
+          this.verify(req, accessToken, refreshToken, profile, verified);
+        } else {
+          this.verify(accessToken, refreshToken, params, profile, verified);
+        };
+      } else { // arity === 4
```

Comment:
I could, but I'm not sure of it's value... if you look at the lib that I'm mimicing, he didn't do it either